code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -4,6 +4,15 @@
 #define POPUP_ANIM_TIME 5
 #define POPDOWN_ANIM_TIME 5 //Be sure to change the icon animation at the same time or it'll look bad
 
+#define TURRET_FLAG_BEEN_ATTACKED		(1<<0)	// The turret gets pissed off and shoots at people nearby (unless they have sec access!)

```

Comment:
I feel like this shouldn't be in this flag list and should stay a bool because it doesn't really fit in with the rest of the checks.

Reply:
Yeah I wasn't too sure about that one while writing this. It basically functions like a temporary TURRET_FLAG_SHOOT_ALL so I decided to include it with the flags since it works like them.

Reply:
This is fine but probably change name to something like SHOOT_ALL_REACTION or similar so it's obvious the effect is the same.